lib/Drush/Commands/OptionsCommands.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Drush\Commands;
+
+/*
+ * Common options providers. Use them by adding an annotation to your method.
+ */
+class OptionsCommands extends DrushCommands {
```

Comment:
If you don't mind mixing hooks with commands, this is a perfectly fine thing to do.

There's nothing wrong with making this extend DrushCommands, but is that necessary? For a commandfile with only hooks, this could perhaps just be `class OptionsCommands`